custom_dc/README.md

```diff
@@ -278,7 +295,8 @@ gcloud run deploy datacommons \
   --image us-central1-docker.pkg.dev/$PROJECT_ID/datacommons/website-compose:$CUSTOM_DC_TAG \
   --add-cloudsql-instances=<project>:<region>:dc-graph \
   --set-env-vars="$env_vars" \
-  --port 8080
+  --port 8080 \
+  --network default
```

Comment:
Can you create a redis instance and deploy a test cloud instance to verify it works as expected?

Reply:
Updated vpc configuration to use a vpc connector.

Working cloud run instance: https://pantheon.corp.google.com/run/detail/us-central1/datacommons-dwnoble/metrics?hl=en&project=datcom-website-dev